index.js

```diff
@@ -1,5 +1,164 @@
 const battleship = () => {
-  return 'The winner is...?'
+
+  // ## Step 1: Create Players
+
+let playerOne = {
+  name: prompt("playerOne - What is your first name?"),
+  shipCount: parseInt(4),
```

Comment:
You don't need to use parseInt if you are declaring it as a number. If you did shipCount: parseInt("4") it would be correct or if was var shipNum = "4" and then you might do shipCount: parseInt(shipNum).